drivers/serial/uart_sam0.c

```diff
@@ -19,6 +19,10 @@
 #define SERCOM_USART_CTRLA_MODE_USART_INT_CLK SERCOM_USART_CTRLA_MODE(0x1)
 #endif
 
+#if defined(SERCOM_U2201) && (REV_SERCOM == 0x500)
+#define SERCOM_REV500
```

Comment:
Maybe add a short comment about REV500. Why we need this?
Is it mandatory use both SERCOM_U2201 and REV_SERCOM flags? Why?

Reply:
SERCOM version (0x102) does not have an interrupt error flag. It is used by SAMD20 and perhaps other devices. Since Atmel headers specify the UART hardware rev and therefore the functionality and available registers I chose to filter by that instead of specific devices to be forward compatible.

I will add a brief comment about this.